src/mono/wasm/build/WasmApp.Common.targets

```diff
@@ -249,6 +249,7 @@
   </PropertyGroup>
 
   <ItemGroup>
+    <NativeFileReference Include="@(NativeLibrary->'%(Identity)')" Condition="'@(NativeLibrary->Count())' != '0'" />
```

Comment:
```suggestion
     <_ExistingNativeLibrary Include="@(NativeLibrary->Exists()) />
    <NativeFileReference Include="@(_ExistingNativeLibrary->'%(Identity)')" />
```

Reply:
Would it make sense to move the condition and items copying into some target executed just before we need these items? So that the lib can be created or copied by some user's target?

Reply:
Moved under `_InitializeCommonProperties` https://github.com/dotnet/runtime/blob/99dd60d8886155bb60768c8ef5a5a3225ad95ba4/src/mono/wasm/build/WasmApp.Common.targets#L275

With this it works when `NativeLibrary` is added under user's target.